lib/main/utils/email_receive_manager.dart

```diff
@@ -5,6 +5,9 @@ import 'package:rxdart/rxdart.dart';
 
 class EmailReceiveManager {
 
+  /// Regex check Url is mail or not
+  final mailRegex = RegExp(r'(([a-zA-Z0-9\-\_\.])+@[a-zA-Z\_]+?(\.[a-zA-Z]{2,6})+)');
```

Comment:
Why don't you use the function `GetUtils.isEmail()` of `Getx` to check if it is `email`?